MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include HISTORY.rst
```

Comment:
FYI turns out not having this blocks installation from the `sdist`. However `whl`s do exist, so `pip` typically prefers those.